syntax = "proto3";
import "c2s_common.proto";

package C2S;


message C2SLoginReq
{
	string sdk_user_id = 1;
	string sdk_token = 2;
	int32 channel_id = 3;
}

message C2SLoginRes
{
	int32 error_code = 1;
	string ip = 2;
	int32 port = 3;
	string profile_id = 4;
}

message C2SLoginZoneReq
{
	string profile_id = 1;
}

message C2SLoginZoneRes
{
	C2S_ERROR_CODE error_code = 1;
}

message C2SKickoutRes
{
	C2S_ERROR_CODE error_code = 1;
}

